client/src/components/CurrentTeam.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import axios from 'axios';
+import ModuleBar from './ModuleBar.jsx';
+import TeamTotal from './TeamTotal.jsx';
+import TeamMember from './TeamMember.jsx'
+
+
+class CurrentTeam extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      people: this.props.test,
+      currentView: this.props.test.slice(0, 8),
+      total: this.props.test.length,
+    }
+  }
+
+  componentWillMount() {
+    this.fetchTeam();
+  }
+
+  fetchTeam() {
+    axios.get('/people/Facebook') //pass :org as a prop and do ${this.props.org}
```

Comment:
I see this is a work in progress. Will you pass in an orgId in the API?

Reply:
endpath with eventually something like this '/people/${this.props.org}'